tests/keras_tests/feature_networks_tests/feature_networks/mixed_precision_tests.py

```diff
@@ -110,13 +111,20 @@ def verify_quantization(self, quantized_model, input_x, weights_layers_idx, weig
                         0] <= unique_tensor_values)
 
         # verify activation quantization
+        holder_layers = get_layers_from_model_by_type(quantized_model, KerasActivationQuantizationHolder)[1:] # skip the input layer
```

Comment:
Why do we skip the input layer? (I believe it's related to the error you were getting with the disconnected graph? can you explain how you managed to solve it?)

Reply:
It was solved by building the model using the wrapped layers (instead of building and wrapping it separately). Even before this change, the input layers were not checked (see the activation indices that were used).